content/spatial/transform.Rmd

```diff
@@ -0,0 +1,54 @@
+---
+title: "Transforming spatial objects"
+date: 2018-02-19T14:52:22+01:00
+---
+
+```{r include = FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = TRUE
+)
+```
+
+
+Sometimes we have a layer in one coordinate system and need to transform it into another coordinate system. The first thing we need to do is identifying both coordinate systems. Let's create an example and identify the coordinates system with `proj4string()`. We used the coordinates posted on the contact page of [NGI](http://www.ngi.be/NL/NL5.shtm).
+
+```{r}
+library(sp)
+library(leaflet)
+```
+
+```{r}
+ngi <- data.frame(x = 650381.78, y = 667603.12)
+coordinates(ngi) <- ~x + y
+proj4string(ngi)
+```
+
+`NA` indicates that the coordinate systems isn't set. So we need to set it manually. In this case we know it is "Lambert 2008". We need to know the related projection string. The projection string is often a rather long string of parameters. However, more projections have an [EPSG](https://en.wikipedia.org/wiki/International_Association_of_Oil_%26_Gas_Producers#European_Petroleum_Survey_Group) number which you can find at http://epsg.io/. The EPSG number for "Lambert 2008" is 3812. Let's set this projection to our dataset. `CRS()` defines the projection based on a text string.
+
```

Comment:
Just to be correct, the term 'coordinate _reference_ system' should be used. A CRS is defined by:

- a coordinate system (like cartesian or polar, with its units)
- the ellipsoid (see `proj -le`), a datum (see `proj -ld`) and prime meridian, that reference the geographic coordinate system (ellipsoid + datum + prime meridian together are often
referred to as 'datum')
- plus for projected CRSes: the conversion formula between geographic and projected CRS (which at the same defines the coordinate system of the projected CRS)